images/macos/provision/core/python.sh

```diff
@@ -7,7 +7,19 @@ echo "Brew Installing Python 3"
 /usr/local/bin/brew install python3
 
 echo "Brew Installing Python 2"
-# Create local tap with formula due to python2 formula depreciation 
+# Create local tap with formula due to python2 formula depreciation
 /usr/local/bin/brew tap-new local/python2
 FORMULA_PATH=$(/usr/local/bin/brew extract python@2 local/python2 | grep "Homebrew/Library/Taps")
 /usr/local/bin/brew install $FORMULA_PATH
+
+echo "Installing pipx"
+export PIPX_BIN_DIR=/opt/pipx_bin
+export PIPX_HOME=/opt/pipx
```

Comment:
May be `/usr/local`? @al-cheb for any thoughts

Reply:
I think `/usr/local` is the better palse, so I moved it to there